config/routes.rb

```diff
@@ -6,10 +6,43 @@
 
   get '/is_it_working' => 'ok_computer/ok_computer#show', defaults: { check: 'default' }
 
-  resources :bulk_actions, except: %i[edit show update] do
+  resources :bulk_actions, only: %i[index new destroy] do
     member do
       get :file
     end
+    scope module: 'bulk_actions' do
+      collection do
+        # In the order they appear in the dropdown
+        resource :apply_apo_defaults_job, only: %i[new create]
+        resource :manage_release_job, only: %i[new create]
```

Comment:
I believe the order may have changed since you create this PR? The order we want is in https://github.com/sul-dlss/argo/pull/3229/files#diff-2e44031d3ab23dbf7a0da286d7b5f23043e6e55fffc309a178a33a58325176a1R17

Reply:
But I'm not sure it matter what order the routes are in... 🤷🏻

Reply:
Thanks, I've updated them to match.